### PR TITLE
deployment/logs-benchmark: add suite for Loki

### PR DESCRIPTION
**Pull request by zekker6**

### CODE REVIEW

File: deployment/logs-benchmark/docker-compose-elk.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  filebeat-vlogs:
+    image: docker.elastic.co/beats/filebeat:8.8.0
+    restart: on-failure
+    volumes:
+      - ./elk/filebeat/filebeat-vlogs.yml:/usr/share/filebeat/filebeat.yml:ro
+    depends_on:
+      - vlogs
+
+  generator:
+    image: golang:1.20-alpine
```
Comment (tenmozes) [inline deployment/logs-benchmark/docker-compose-elk.yml:21]: 1.21.3?

**Review by tenmozes — CHANGES_REQUESTED** [review]
We need to update https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/deployment/logs-benchmark#how-to-run and add ELK and Loki sections

**Comment by zekker6** [comment]
Isn't it covered in step 4 of "how to run"?

File: deployment/logs-benchmark/readme.md [hunk 2]
```diff
@@ -1,23 +1,44 @@
 # Benchmark for VictoriaLogs
 
+Benchmark compares VictoriaLogs with ELK stack and Grafana Loki.
```
Comment (tenmozes) [inline deployment/logs-benchmark/readme.md:3]: Can we be consistent with the docs, have general readme with loki and elk or have a readme per benchmark
I recommend to keep everything in one readme as far as we have only elk and loki

**Review by tenmozes — APPROVED** [review]

**Pull request merged**